livecheck/special/sourceforge.py

```diff
@@ -11,18 +12,23 @@ def get_latest_sourceforge_package(project_name: str) -> tuple[str, str]:
     try:
         response = requests.get(url)
         response.raise_for_status()
-
-        release_json = response.json()
-
-        filename = release_json['release']['filename']
-
-        version_match = re.search(r'/(\d+\.\d+\.\d+)/', filename)
-
-        if version_match:
-            version = version_match.group(1)
-            download_url = f"https://downloads.sourceforge.net/{project_name}/{project_name}-{version}.tar.gz"
-            return version, download_url
-
+        if response.status_code == HTTPStatus.OK:
```

Comment:
This line won't hit because of the above line `response.raise_for_status()` which raises an exception if the status is not 200.